android/src/com/mapswithme/maps/widget/placepage/PlacePageView.java

```diff
@@ -703,6 +710,22 @@ void refreshViews()
     refreshPreview(mMapObject);
     refreshDetails(mMapObject);
     refreshViewsInternal(mMapObject);
+
+    //
+    // The view is completely broken after the first call to refreshView():
+    // https://github.com/organicmaps/organicmaps/issues/722
+    // https://github.com/organicmaps/organicmaps/issues/1065
+    //
+    // Force re-layout explicitly on the next event loop after the first call to refreshView().
+    //
+    if (!mIsRefreshViewCalled) {
```

Comment:
I'd prefer to add additional condition **if (Build.VERSION.SDK_INT >= 30)** because of it reproduces nly on Android 11+ devices.

Reply:
Agree. This whole relayout thing should be under if